conf/app.routes

```diff
@@ -9,3 +9,6 @@ POST        /declarations/sendEmails/:id        uk.gov.hmrc.merchandiseinbaggage
 POST        /calculation                        uk.gov.hmrc.merchandiseinbaggage.controllers.CalculationController.handleCalculation()
 
 GET         /payment-callback/:mibRef           uk.gov.hmrc.merchandiseinbaggage.controllers.DeclarationController.paymentSuccessCallback(mibRef: String)
+
+GET         /validate/eori/:eoriNumber          uk.gov.hmrc.merchandiseinbaggage.controllers.EoriCheckNumberController.checkEoriNumber(eoriNumber: String)
```

Comment:
the url can be  `/eori/:eoriNumber` because you are not actually validating anything but just returning the eori details to the caller

Reply:
i copied from you slack :-)
Suresh Mandalapu  2 hours ago
GET    /validate/eori/:eoriNumber

Reply:
BTW we call an API to do the validation and the response contains `valid` field as well.....so what you and Jordan suggested /validate/eori/:eoriNumber first time looks good to me......

Reply:
ohh this one,  i mean to say a GET url with 200/404 response ia a pattern, not the exact url string :) 
sorry if i misled you, but we can keep `/validate` if you want to, as it's just our service